<ui version="4.0" >
 <class>CostDialogCard</class>
 <widget class="QDialog" name="CostDialogCard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/resources.qrc" >:/32/money.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>215</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Cost:*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="costlineEdit" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Catring:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QComboBox" name="cartingcomboBox" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addpushButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelpushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cartingcomboBox</tabstop>
  <tabstop>costlineEdit</tabstop>
  <tabstop>addpushButton</tabstop>
  <tabstop>cancelpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>CostDialogCard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>277</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>304</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
